trunk/SUAVE/Plots/Geometry_Plots/plot_polars.py

```diff
@@ -0,0 +1,88 @@
+## @ingroup Plots-Geometry_Plots
+# plot_polars.py
```

Comment:
Looks like you haven't added this file to the regressions:
https://coveralls.io/builds/40138905/source?filename=%2Fhome%2Fappveyor%2Fprojects%2Fsuave%2Ftrunk%2FSUAVE%2FPlots%2FGeometry_Plots%2Fplot_polars.py

Reply:
Fixed